app/models/response.rb

```diff
@@ -9,6 +9,8 @@ class Response < ApplicationRecord
   validates :answers, presence: true
 
   scope :unarchived, -> { where(archived: false) }
+  scope :summative, -> { where('schema->>1 = ?', 'summative_questionnaire') }
```

Comment:
The holdover from the previous version I think is worth addressing is the mismatch between typing variable names both in the DB, CMS and code base.

I had introduced `Training::Question#assessments_type` as a way to start the transition. I think these two scopes might make it harder for us.

`UserAnswers#assessments_type` suggestions there are three flavours of assessment. I think what was intended is to record the question type instead. 

In an ideal world we would be able to also rename the question page types in Contentful removing the `_questionnaire` suffix.

At a glance though these scopes are approaching the filter using different groups of type.

Assessment types:
1. `summative_assessment` (exists in the business logic and is the only type in our assessment table)
2. `formative_assessment` (never used)
3. `confidence_check` (never used)

Question types: (the final hiding place of the word "questionnaire")
1. `summative_questionnaire` (only type of question found in an assessment)
2. `formative_questionnaire`
3. `confidence_questionnaire`